runtime/src/main/java/org/corfudb/runtime/collections/CorfuTable.java

```diff
@@ -186,9 +183,9 @@ public int hashCode() {
      * @param <V>           Value type
      * @return              A type token to pass to the builder.
      */
-    static <K, V> TypeToken<CorfuTable<K, V>>
-        getMapType() {
-            return new TypeToken<CorfuTable<K, V>>() {};
+    public static <K, V> TypeToken<CorfuTable<K, V>> getMapType() {
+        return new TypeToken<CorfuTable<K, V>>() {
+        };
```

Comment:
Can you please revert this file. It will cause bunch of conflicts with my changes.